.github/release.yml

```diff
@@ -0,0 +1,22 @@
+# release.yml
+
+changelog:
+  exclude:
+    labels:
+      - dependencies
+      - chore
+    authors:
+      - dependabot
+  categories:
+    - title: 🚧 Breaking Changes
+      labels:
+        - breaking-change
```

Comment:
I think we use the unstable [label](https://github.com/Shopify/ruby-lsp/labels) for this one.

Reply:
I think `unstable` and `breaking-change` means different things? According to the new release strategy, `unstable` means the feature will be guarded behind feature flags and are hidden by default. So it should actually be safe to release.

Breaking changes, on the other hand, are expected to break users usages or expectations. This may be rare for this project though and I can remove this part for now to avoid confusion.